nemo/collections/asr/models/asr_model.py

```diff
@@ -142,6 +142,10 @@ def on_after_backward(self):
                 logging.warning(f'detected inf or nan values in gradients! Setting gradients to zero.')
                 self.zero_grad()
 
+    @property
```

Comment:
Put this in the insivyao classes, not in the base class. Not all sunglasses are ASR

Reply:
Plus add a setter property in those classes too

Reply:
Should there be an implementation check in base model to enforce?